File: src/main/java/de/thatsich/intellie/common/module/block/IBlockContainer.java
package de.thatsich.intellie.common.module.block;

import de.thatsich.intellie.common.module.item.AItemBlock;
import net.minecraft.block.ITileEntityProvider;

/**
 @author thatsIch
 @since 06.03.14. */
public interface IBlockContainer extends IBlock, ITileEntityProvider
{
	Class<? extends AItemBlock> getItemBlockClass();
}

File: src/main/scala/de/thatsich/minecraft/intellie/common/module/block/IBlockContainer.scala
package de.thatsich.minecraft.intellie.common.module.block

import net.minecraft.block.ITileEntityProvider

/**
 *
 *
 * @author thatsIch
 * @since 07.04.2014.
 */
trait IBlockContainer extends IBlock with ITileEntityProvider
{
	def getItemBlockClass: Class[ _ => IItemBlock ]
}
